.github/workflows/continous-integration.yml

```diff
@@ -76,6 +76,9 @@ jobs:
     - name: Install Dependencies 📦
       run: |
         sudo apt-get -y install libpq-dev
+        cp poetry.lock poetry.lock.old
+        poetry update
```

Comment:
do we really need to do this? it takes quite some time

Reply:
no, this is just for debugging 😄 The build keeps [failing](https://github.com/RasaHQ/rasa/runs/735007342?check_suite_focus=true#step:7:51) and I'm trying to figure out why